management/prompt.go

```diff
@@ -30,3 +36,39 @@ func (m *PromptManager) Read(opts ...RequestOption) (p *Prompt, err error) {
 func (m *PromptManager) Update(p *Prompt, opts ...RequestOption) error {
 	return m.Request("PATCH", m.URI("prompts"), p, opts...)
 }
+
+// MarshalJSON is a custom serializer for the promptCustomText type.
+//
+// We have to use one to avoid creating a new Request method that does not JSON-encode the request body.
+func (t *promptCustomText) MarshalJSON() ([]byte, error) {
+	return []byte(auth0.StringValue(t.Text)), nil
+}
+
+// UnmarshalJSON is a custom deserializer for the promptCustomText type.
+//
+// We have to use one to avoid creating a new Request method that does not JSON-decode the response body.
+func (t *promptCustomText) UnmarshalJSON(b []byte) error {
+	t.Text = auth0.String(string(b))
+	return nil
+}
+
+// Retrieve custom text for a specific prompt and language.
+//
+// See: https://auth0.com/docs/api/management/v2#!/Prompts/get_custom_text_by_language
+func (m *PromptManager) CustomText(p string, l string, opts ...RequestOption) (t *string, err error) {
+	var r *promptCustomText
+	err = m.Request("GET", m.URI("prompts", p, "custom-text", l), &r, opts...)
```

Comment:
```suggestion
	err = m.Request(http.MethodGet, m.URI("prompts", p, "custom-text", l), &r, opts...)
```

What do you think about using the constants provided by the http pkg?

Reply:
The project's convention seems to be to use strings:
- https://github.com/go-auth0/auth0/blob/master/management/client.go#L176
- https://github.com/go-auth0/auth0/blob/master/management/actions.go#L171
- https://github.com/go-auth0/auth0/blob/master/management/branding.go#L138
- ...
However, if you think we should use the constants anyway, I'll commit your suggestion.

Reply:
Thanks for pointing that out:) let's keep with the convention for now then as it will be easier for contributors.